neura/assetlist.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "neura",
+  "assets": [
+    {
+      "description": "ANKR: The native EVM, governance, and staking token for Neura, enabling secure transactions, and seamless GPU resourcing within the ecosystem.",
+      "denom_units": [
+        {
+            "denom": "atankr",
+            "exponent": 0
+        },
+        {
+            "denom": "ankr",
+            "exponent": 18
+        }
+      ],
+      "base": "atankr",
+      "name": "Neura",
+      "display": "ankr",
+      "symbol": "ANKR",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neura/images/ankr.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neura/images/ankr.svg"
+        }
+      ],
+      "coingecko_id": "ankr-network",
```

Comment:
Is this supposed to be the same token as ANKR on Ethereum?
If so, because they have different places of origin, we have to treat them as different assets. If we want to be able to define the coingecko-id for ANKR here, we should register ANKR to Ethereum's assetlist, and then define this asset (Neura::ANKR) as an `additional-mintage` of Ethereum::ANKR

Reply:
Yes, it's the same token